README.md

```diff
@@ -25,14 +25,14 @@ If you need help using or installing Node.js, please use the
 * [Contributing to the project](./CONTRIBUTING.md)
 * IRC (node core development): [#node-dev on chat.freenode.net](https://webchat.freenode.net?channels=node-dev&uio=d4)
 
-### Community Resources
+### Unofficial Resources
 
 * IRC (general questions): [#node.js on chat.freenode.net](https://webchat.freenode.net?channels=node.js&uio=d4). Please
 see http://nodeirc.info/ for more information regarding the `#node.js` IRC channel.
 
-*Please note that community provided resources are neither managed by -- nor
-necessarily endorsed by -- the Node.js TSC/CTC. Specifically, such resources are
-not currently covered by the [Node.js Moderation Policy][] and the selection and
+*Please note that unofficial resources are neither managed by -- nor necessarily
+endorsed by -- the Node.js TSC/CTC. Specifically, such resources are not
```

Comment:
I guess one obvious question is why `TSC/CTC` as opposed to one or the other? Shouldn't one or the other be managing the official IRC channels instead of being fuzzy about it?